### PR TITLE
Use tito for releases

### PR DESCRIPTION
**Pull request by MartinBasti**
Make releases automated as much as possible

Signed-off-by: Martin Bašti <mbasti@redhat.com>

Maintainers will complete the following section:
- [ ] Commit messages are descriptive enough
- [ ] "Signed-off-by:" line is present in each commit
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] JSON/YAML configuration changes are updated in the relevant schema
- [ ] Changes to metadata also update the documentation for the metadata
- [ ] Pull request includes link to an osbs-docs PR for user documentation updates

### CODE REVIEW

**Comment by MartinBasti** [comment]
@StykMartin :eyes:

File: osbs-client.spec [hunk 1]
```diff
@@ -32,20 +29,17 @@
...
-%endif
+Version:        0.64.1
+Release:        1%{?dist}
 
 Summary:        Python command line client for OpenShift Build Service
 Group:          Development/Tools
 License:        BSD
 URL:            https://github.com/containerbuildsystem/osbs-client
-Source0:        https://github.com/containerbuildsystem/osbs-client/archive/%{commit}/osbs-client-%{commit}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
```
Comment (StykMartin) [inline osbs-client.spec]: ugh. This should point to an upstream resource for your tarball. 
The goal is to separate upstream source and vendor modifications.

File: setup.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ def _install_requirements():
 setup(
     name="osbs-client",
     description='Python module and command line client for OpenShift Build Service',
-    version="0.65.dev",
+    version="0.64.1",
```
Comment (StykMartin) [inline setup.py]: So you keeping here version, which is already released in PyPI.
What will happen if someone will try to use dev and version is same as released one

Comment (MartinBasti) [reply setup.py]: Hmm, while I'm thinking about this, probably we should just follow pattern from https://github.com/containerbuildsystem/koji-containerbuild and don't bump to dev version. It makes tito workflow easy. Dev version must be force installed.

File: osbs-client.spec [hunk 3]
```diff
@@ -145,7 +127,7 @@ This package contains osbs Python 2 bindings.
 
 
 %prep
-%setup -qn %{name}-%{commit}
+%setup -q
```
Comment (athos-ribeiro) [inline osbs-client.spec]: Could we use autosetup here? (nitpick)

https://rpm.org/user_doc/autosetup.html

File: setup.py (hunk 2 above)
Comment (athos-ribeiro) [reply setup.py]: Could we fetch this version from a single place, such as osbs/version.py?

File: osbs-client.spec (hunk 3 above)
Comment (MartinBasti) [reply osbs-client.spec]: I;d do it in separate PRs, I;d like to keep it in sync with other projects

File: setup.py (hunk 2 above)
Comment (MartinBasti) [reply setup.py]: We could, but it;s not nice and tito handles this for us

**Comment by MartinBasti** [comment]
No release notes, change to release process only

**Pull request merged**